src/routes/index.ts

```diff
@@ -10,4 +10,12 @@ router.get('/character', function(req, res, next) {
   res.render('character', { testejs: 'hi!' });
 });
 
+router.get('/login', function(req, res, next) {
+  res.render('loginView', { testejs: 'hi!' });
```

Comment:
do you actually use the passed object within EJS template? if not, then just: `res.render('loginView');`